ps_spec.yml

```diff
@@ -0,0 +1,49 @@
+version: 1.0
+service_name: cwe-collector
+
+stages:
+    -
+        name: PR Test
+        when:
+            - pull_request
+            - pull_request:
+                trigger_phrase: test it
+        image: node:12
+        compute_size: small
+        commands:
+            - make test
+
+    -
+        name: Master Push - Publish
```

Comment:
we don't need this step because we don't publish collector packages in npm

Reply:
okay. I will update.